src/components/a-painter.js

```diff
@@ -1,7 +1,12 @@
-/* global AFRAME saveAs Blob uploadcare */
+/* global AFRAME Blob uploadcare */
+
+var saveAs = require('../../vendor/saveas.js').saveAs;
+
 AFRAME.APAINTER = {
-  version: 1,
-  brushes: {},
+  version: 1
+};
+
+AFRAME.registerComponent('a-painter', {
```

Comment:
Just realize it'd make more sense as a system, and to name `painter` so we can do `sceneEl.systems.painter`. Don't actually have to change much when we do this.

`AFRAME.registerSystem('painter')`